eng/pipelines/azure-pipelines.yml

```diff
@@ -98,7 +98,7 @@ stages:
       jobs:
 
       - job: windows
-        timeoutInMinutes: 30
+        timeoutInMinutes: 45
```

Comment:
Can we just up the timeout when we are actually signing? I'm not sure it makes sense to increase the timeouts on PR run.

Reply:
sure, we could. Are we still worried about hanging tests? Is that still happening? Also, do note that this is only being extended in release branch which is where we sign for official builds

Reply:
> Are we still worried about hanging tests? Is that still happening?

Yes, I've seen it a couple times lately. I think it is the SqlServer tests, but I'm not sure exactly which one. I've looked at all the "starting" and "finished" lines in a hang, and SqlServer is the only one started that didn't finish.

> Also, do note that this is only being extended in release branch which is where we sign for official builds

I assume you will need to replicate it for each new release branch then?

Reply:
Yes, but we already have sets of changes that need to be made for each release branch, like enabling signing for instance.